client/src/components/EditReviewForm.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorList"
+import translateServerErrors from "../services/translateServerErrors"
+
+const EditReviewForm = (props) => {
+  const [editReview, setEditReview] = useState({
+    body: "",
+    score: "",
+    userId: "",
+    reviewId: "",
+    showId: ""
+  })
+
+  const [errors, setErrors] = useState([])
+
+  const userId = props.userId
+
+  const handleInputChange = (event) => {
+    setEditReview({
+      ...editReview,
+      [event.currentTarget.name]: event.currentTarget.value,
+      userId: userId,
+      reviewId: props.reviewId,
+      showId: props.showId
+    })
+  }
+
+  const showId = props.showId
+
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+    props.getShow()
```

Comment:
we'll want to now remove this `props.getShow()` since a successful fetch request should now re-render the page